Ska/DBI/DBI.py

```diff
@@ -91,7 +91,7 @@ def __init__(self, dbi=None, server=None, user=None, passwd=None, database=None,
                 raise ValueError(
                     'SYBASE_OCS env variable not defined. sybase dbi works only in production ska.')
             modulepath = os.path.join(os.environ['SYBASE'], os.environ['SYBASE_OCS'],
-                                      'python', 'python34_64r', 'lib')
+                                      'python', 'python39_64r', 'lib')
```

Comment:
For this, I could do something to check the python version and then select the right directory, but I figured we didn't need updates to this to be back-compatible with older Python versions.

Reply:
Sounds fine.

Reply:
we would want it to be compatible with the current version, right? Will this PR work with the current flight version?

Reply:
I don't think we necessarily need it to be compatible with the current python version, as we don't do "mixed' environments (ska3-flight at for Python 3.10.8 doesn't have an option to work with a different ska3-core) and we can control this by just promoting this change with ska3-prime.  If we are actually worried about it, I can add the logic.

Reply:
I think @jeanconn is proposing that this PR be included only in the ska3-prime release. We generally don't require those packages to be back-compatible with earlier Python versions.

Reply:
ok, but as soon as this is merged masters tests will be broken until we have a ska3-prime release.

Reply:
Yes.  Along those lines though, does it make sense to update masters testing to work from a ska3 prime candidate after 2022.13 is promoted anyway?

Reply:
I suppose we should.